cmd/admin.go

```diff
@@ -155,6 +155,7 @@ var (
 	microcmdAuthDelete = cli.Command{
 		Name:   "delete",
 		Usage:  "Delete specific auth source",
+		Flags:  []cli.Flag{idFlag},
```

Comment:
What's the purpose of `Flags:` entry?

Reply:
The error was because id was not specified as a possible flag for "delete" command thus commands such as `./gitea admin auth delete --id 4` resulted in `Incorrect Usage: flag provided but not defined: -id` the method for handling deletion of `LoginSource` already depends upon id being provided through the `--id` flag. But it was not defined in `microcmdAuthDelete` adding the `Flags:  []cli.Flag{idFlag}`  makes id to be a possible flag for `./gitea admin auth delete` command.